### ISSUE TITLE
Redox

### ISSUE THREAD

**Issue by naturallymitchell** [dirs-dev/dirs-rs#11]

**Comment by soc** [comment]
Hi @mitchtbaum, what changes do you suggest?

**Comment by naturallymitchell** [comment]
this would seem worthwhile to work on Redox, in addition to Lin, Win, and Mac.

but actually, https://github.com/soc/dirs-rs/issues/12 is much better to look into first.

**Comment by soc** [comment]
As far as I see, Redox is already supported. Anything that is missing there?

**Comment by naturallymitchell** [comment]
I haven't tried it, so hard to say.

If you could add that info to the Readme, we could help push it along a bit.

**Comment by soc** [comment]
It's the second operating system mentioned in the README, after Linux and before Windows and macOS.

**Comment by naturallymitchell** [comment]
Okay.  Duh.

I brushed by it and looked mainly at the Examples and Features table.  The features table using a slash after Linux is hard to parse accurately+quickly, imho.  What do you think about changing it to an "and"?  And If the examples could also include "Redox", that'd be good.  What do you think?

**Comment by naturallymitchell** [comment]
also the project's description line

**Comment by archseer** [comment]
Would be nice if there was a smaller crate than `redox_users` to depend to, since `redox_users` depends on `argon2`, `failure`, `failure-derive`, `rand_os` and a bunch of other sub-dependencies.

I know it's only included if built for redox, but still.

**Comment by soc** [comment]
I'd close this, `dirs-sys-rs` is using `redox_users = {default-features = false }` for a while now, which seems to avoid a lot of dependencies.

Feel free to reopen!
